src/main/java/applemazer/Ui.java

```diff
@@ -0,0 +1,85 @@
+package applemazer;
+
+import tasks.Task;
+
+/**
+ * Class that handles chatbot messages.
+ */
+public class Ui {
+    /**
+     * Constructor for the Ui object.
+     */
+    public Ui() {}
```

Comment:
Empty constructor is present by default in Java. If it is the only constructor, it isn't required, though not wrong if it is there as well.